batch_update_users.py

```diff
@@ -141,6 +141,15 @@ def main():
                 member_type = None
         except KeyError:
                 member_type = None
+        try:
+            if len(m['isArchived']) > 1 and m['isArchived'] == "True":
+                member_isArchived = True
+            elif len(m['isArchived']) > 1 and m['isArchived'] == "False":
+                member_isArchived = False
+            else:
+                member_isArchived = None
+        except KeyError:
+                member_isArchived = None
```

Comment:
Try this instead? 

```suggestion
            archived_value = m.get('isArchived', None)
            if archived_value == "True":
                member_isArchived = True
            elif archived_value == "False":
                member_isArchived = False
            else:
                member_isArchived = None
```

This way we don't rely on the error type to set `member_isArchived=None`